src/Command/ArchiveCertainCoinsPrices.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace App\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use App\Service\Coins\ArchivePrices;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputArgument;
+
+/**
+ * Class ArchiveCertainCoinsPrices
+ */
+#[AsCommand(
+    name: 'prices:list:archive',
+    description:'Saves coins prices to archive'
+)]
+class ArchiveCertainCoinsPrices extends Command
+{
+    /**
+     * @param ArchivePrices $archivePrices
+     */
+    public function __construct(
+        private ArchivePrices $archivePrices
+        )
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        parent::configure();
+        $this->setName('prices:archive')
```

Comment:
set name and description in redundant because this is set as php attribute.
The same in all command.